MiniApp/Classes/core/JavascriptBridge/MiniAppScriptMessageHandler.swift

```diff
@@ -464,6 +466,20 @@ internal class MiniAppScriptMessageHandler: NSObject, WKScriptMessageHandler {
         }
     }
 
+    func fetchPoints(with callbackId: String) {
+         if isUserAllowedPermission(customPermissionType: .points, callbackId: callbackId) {
+            hostAppMessageDelegate?.getPoints { (result) in
+                switch result {
+                case .success(let response):
+                    guard let encodedPoints = ResponseEncoder.encode(data: response) else { return }
+                    self.executeJavaScriptCallback(responseStatus: .onSuccess, messageId: callbackId, response: encodedPoints)
+                case .failure(let error):
+                    self.handleMASDKError(error: error, callbackId: callbackId)
```

Comment:
Please call [prepareMAJavascriptError](https://github.com/rakutentech/ios-miniapp/blob/master/MiniApp/Classes/core/JavascriptBridge/MAJavascriptErrorHandler.swift#L17) to make sure we follow the same error format.